### PR TITLE
Cognito idp revoke token

### PR DESCRIPTION
**Pull request by maxon755**
depends on https://github.com/async-aws/aws/pull/1295

### CODE REVIEW

File: src/Core/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
 
 ## NOT RELEASED
 
+## 1.18.0
```
Comment (jderusse) [inline src/Core/CHANGELOG.md:5]: thanks for updating the changelog, but this is not how it works.

- you should not add the version (just add a `### Added` section under `## NOT RELEASED`)
- You should update the `CHANGELOG.md` inside `src/Service/CognitoIdentityProvider/` folder
- try to be consistent with other changes (You'll find examples is the file above) like 
```
Added operation `AdminAddUserToGroup`
Added operation `AdminRemoveUserFromGroup`
...
```


thanks for you help

Comment (maxon755) [reply src/Core/CHANGELOG.md]: Thanks for the explanation. Updated

File: src/Core/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
 # Change Log
 
 ## NOT RELEASED
-
```
Comment (jderusse) [inline src/Core/CHANGELOG.md:4]: please revert

Comment (maxon755) [reply src/Core/CHANGELOG.md]: done

**Comment by maxon755** [comment]
Hey, @jderusse! What about this PR?

**Comment by jderusse** [comment]
thank you @maxon755

**Comment by maxon755** [comment]
Thank you for your great project, @jderusse !
When my changes may be released?

**Comment by jderusse** [comment]
1.6.0 released

**Pull request merged**